docs/design/chart_format.md

```diff
@@ -412,7 +304,13 @@ A generic short form maps to an object storage reference (`DOMAIN/REPOSITORY/REL
 helm:example.com/charts/nginx-1.2.3.tgz
 ```
 
-For the purpose of providing versioning ranges, and also for backward compatibility, version requirements may be treated as as suffix condition, instead of part of the path:
+Or, in the case of the Google Storage (`gs://`) and the S3 scheme (`s3://`), the domain indicates the storage scheme.
+
+```
+helm:gs/kubernetes-charts/nginx-1.2.3.tgz
+```
+
+For the purpose of providing versioning ranges, and also for backward compatibility, version requirements may be treated as a suffix condition, instead of as part of the path:
```

Comment:
Unless I'm mistaken, they MUST be treated as suffix conditions in short references. We do not recognize version information in the path (e.g., in the chart name or in an earlier path segment), and including it there will cause a parse error and/or a name matching failure.